src/formatDate.js

```diff
@@ -51,6 +51,40 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const newDate = [0, 0, 0];
+  const dateSplit = date.split(fromFormat[3]);
+  const oldDayIndex = fromFormat.indexOf('DD');
+  const oldMonthIndex = fromFormat.indexOf('MM');
+  const DayIndex = toFormat.indexOf('DD');
+  const MonthIndex = toFormat.indexOf('MM');
```

Comment:
why did you name it with uppercase letter? You should name variables starting with lowercase letter

Reply:
sorry i made a copy from old and forgot to lower case them